src/main/webapp/components/utils/getField.js

```diff
@@ -0,0 +1,13 @@
+/**
+  * Returns the value at obj[field] or defaultValue
```

Comment:
Documentation is not supposed to repeat the code, but add value on top of a short description.

Reply:
Updated the comment to be less descriptive and to be more use case orientated. Commited with f764c5c